src/pages/form-validation/list/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useCallback, useState } from 'react';
+import {
+    AppBar,
+    Card,
+    Hidden,
+    IconButton,
+    InputProps,
+    Switch,
+    TextField,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import { Dns, Menu, Timeline } from '@material-ui/icons';
+import List from '@material-ui/core/List';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { InfoListItem } from '@pxblue/react-components';
+
+type OnChangeHandler = InputProps['onChange'];
+
+const useStyles = makeStyles((theme: Theme) => ({
+    containerWrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'start',
+        flex: '1 1 0',
+    },
+    container: {
+        maxWidth: 686,
+        margin: theme.spacing(4),
```

Comment:
`margin-top` should be 24 so you can just do this:
```suggestion
        margin: theme.spacing(3),
```